src/ECU.cpp

```diff
@@ -3,20 +3,17 @@
 #include "Throttle.h"
 #include "Brake.h"
```

Comment:
Needs #include <Arduino.h>

Reply:
It doesn't. Tested without it in Aug and works

Reply:
That's because FlexCAN_T4 includes Arduino.h. Same concept as why we need to include FlexCAN_T4 in the lib_deps, we should theoretically explicitly include <Arduino.h> here. Up to you, but there is no harm in adding it